<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_menu</class>
 <widget class="QWidget" name="login_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>518</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI Light</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>500</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>398</width>
        <height>498</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="username_box">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>200</y>
            <width>150</width>
            <height>27</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_box">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>240</y>
            <width>150</width>
            <height>27</height>
           </rect>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>200</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>240</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="login_button">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>300</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>390</y>
            <width>150</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Don't have account?</string>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="register_button">
          <property name="geometry">
           <rect>
            <x>235</x>
            <y>390</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Click Here</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
